TADashboard/src/main/java/page_objects/DashboardPage.java

```diff
@@ -1,11 +1,19 @@
 package page_objects;
 
 import com.logigear.control.common.imp.Label;
+import com.logigear.control.common.imp.Link;
 
 public class DashboardPage {
+    private Link lnkDataProfiles = new Link("css=li>ul>li>a[href='profiles.jsp']");
+    private Link lnkAdminister = new Link("css=a[href='#Administer']");
     private Label lblWelcome = new Label("css=a[href='#Welcome']");
 
     public boolean doesWelcomeLabelDisplay() {
         return lblWelcome.isVisible();
     }
-}
+
+    public void clickDataProfilesLink() {
```

Comment:
you should have a better method, the name is just click Profiles but you also click Administer as well.

Reply:
Yeah, I fixed it. Please check again.